plaso/preprocessors/linux.py

```diff
@@ -237,7 +237,7 @@ def _ParseFileData(self, knowledge_base, file_object):
     line_reader = line_reader_file.BinaryLineReader(file_object)
 
     try:
-      reader = csv.reader(line_reader, delimiter=b':')
+      reader = line_reader_file.BinaryDSVReader(line_reader, delimiter=b':')
```

Comment:
Delimiter is not a kwarg, update the caller or the class method

Reply:
Done